### ISSUE TITLE
Problem while installing version 0.17.4

### ISSUE THREAD

**Issue by Sharpek** [deschler/django-modeltranslation#624]
Hi,

I think in latest release (0.17.4) is a bug. I could not install it in my application. Here you have traceback:

```
╰─$ pip install  django-modeltranslation==0.17.4
Collecting django-modeltranslation==0.17.4
  Using cached django-modeltranslation-0.17.4.tar.gz (70 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  ERROR: Command errored out with exit status 1:
   command: /home/mbaran/.virtualenvs/booking/bin/python /home/mbaran/.virtualenvs/booking/lib/python3.10/site-packages/pip/_vendor/pep517/in_process/_in_process.py get_requires_for_build_wheel /tmp/tmpx74ffx8x
       cwd: /tmp/pip-install-ur59yoh1/django-modeltranslation_543ef6de278e4f8cab484792165adcaf
  Complete output (20 lines):
  Traceback (most recent call last):
    File "/home/mbaran/.virtualenvs/booking/lib/python3.10/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 363, in <module>
      main()
    File "/home/mbaran/.virtualenvs/booking/lib/python3.10/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 345, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "/home/mbaran/.virtualenvs/booking/lib/python3.10/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 130, in get_requires_for_build_wheel
      return hook(config_settings)
    File "/tmp/pip-build-env-cz317vp1/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 162, in get_requires_for_build_wheel
      return self._get_build_requires(
    File "/tmp/pip-build-env-cz317vp1/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 143, in _get_build_requires
      self.run_setup()
    File "/tmp/pip-build-env-cz317vp1/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 267, in run_setup
      super(_BuildMetaLegacyBackend,
    File "/tmp/pip-build-env-cz317vp1/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 158, in run_setup
      exec(compile(code, __file__, 'exec'), locals())
    File "setup.py", line 5, in <module>
      version = __import__('modeltranslation').get_version()
    File "/tmp/pip-install-ur59yoh1/django-modeltranslation_543ef6de278e4f8cab484792165adcaf/modeltranslation/__init__.py", line 6, in <module>
      import django
  ModuleNotFoundError: No module named 'django'
  ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/ad/70/041ea35a3e1dd00deae4c4fee189226ab0415c7b7b83c7801901348e01c4/django-modeltranslation-0.17.4.tar.gz#sha256=c53e75d44438fdd768f8eb0197b3a4cbda3b581e5d4c59c080132af54ebe5c08 (from https://pypi.org/simple/django-modeltranslation/). Command errored out with exit status 1: /home/mbaran/.virtualenvs/booking/bin/python /home/mbaran/.virtualenvs/booking/lib/python3.10/site-packages/pip/_vendor/pep517/in_process/_in_process.py get_requires_for_build_wheel /tmp/tmpx74ffx8x Check the logs for full command output.
ERROR: Could not find a version that satisfies the requirement django-modeltranslation==0.17.4 (from versions: 0.2, 0.3, 0.3.1, 0.3.2, 0.3.3, 0.4.0a1, 0.4.0b1, 0.4.0b2, 0.4.1, 0.5a1, 0.5b1, 0.5, 0.5.1, 0.6, 0.6.1, 0.7, 0.7.1, 0.7.2, 0.7.3, 0.8b1, 0.8b2, 0.8, 0.8.1, 0.9, 0.9.1, 0.10, 0.10.1, 0.10.2, 0.11rc1, 0.11rc2, 0.11, 0.12, 0.12.1, 0.12.2, 0.13b1, 0.13b2, 0.13b3, 0.13, 0.13.1, 0.13.2, 0.13.3, 0.13.4, 0.14, 0.14.1, 0.14.2, 0.14.3, 0.14.4, 0.15, 0.15.1, 0.15.2, 0.16, 0.16.1, 0.16.2, 0.17, 0.17.1, 0.17.2, 0.17.3, 0.17.4)
ERROR: No matching distribution found for django-modeltranslation==0.17.4
```

**Comment by last-partizan** [comment]
That's strange, i can reproduce it.

**Comment by last-partizan** [comment]
Fixed in 0.17.5
